brackend/db/models.py

```diff
@@ -41,6 +50,7 @@ class Tournament(Base):
     id = Column(Integer, primary_key=True)
     name = Column(String(255), nullable=False)
     date = Column(DateTime, default=datetime.now)
+    game = Column(String(255))
```

Comment:
I was thinking we'd use an Enum here so that we could have a specific list of games to support. I guess we can do this by having a list to select from on the fontend

Reply:
okay, how would that look? would the enum have titles mapped to specific values? something like
class Games:
     melee = enum.auto()
?
or would it map to the title?

Reply:
Yeah, that's what I was thinking, but honestly for now it's totally fine to keep as a string, because we don't really know whether this is going to be required, and we don't need to worry about space being used by the DB. So you can disregard this for now, and we can just provide a drop down on the frontend